omigami/data/__init__.py

```diff
@@ -0,0 +1,21 @@
+from omigami.data.data_types import (
+    NumpyArray,
+    Estimator,
```

Comment:
Is Estimator really a data type or rather a interface?

Reply:
there is the `InputEstimator` (now renamed) and the `Estimator` interface